ail_vru_uc1_scenario/cdasim_config/run_simulation

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Script to start simulation
+# Launches RViz
+# Prunes all previous Kafka volumes
+carma stop all
+
+export ROS_MASTER_URI=http://172.3.0.2:11311/
+
+bash -c 'sleep 35 && rviz -d ~/carma_ws/src/carma-platform/carma/rviz/carma_default.rviz' &
+
+docker volume rm \
+  carma_kafka-datavolume \
+  carma_zookeeper-datavolume
```

Comment:
We should add a comment explaining why we have to explicitly delete these volumes

Reply:
Resolved